### PR TITLE
User/hshami/time series

### PR DESCRIPTION
**Pull request by mhshami01**
- update tracker to upload telemetry for time series analyses.
- added skeleton perf test scripts.
- added skeleton perf pipeline.

### CODE REVIEW

**Review by marianan — APPROVED** [review]
:shipit:

File: tests/eh_send_one_message.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
```
[the remark this answers is not in the document]
Comment (mhshami01) [reply tests/eh_send_one_message.py:1]: There are two files/python scripts that do the same thing:
- eh will send one message to an event hub as specified in the command line, and
- ih will send one message to an IoT device as specified in the command line.

the eh is not used and will be moved to a different branch for later potential use. the ih file will be renamed to 'send_one_message_to_iot_hub_device.py

File: tests/eh_send_one_message.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
```
[the remark this answers is not in the document]
Comment (mhshami01) [reply tests/eh_send_one_message.py:3]: spoke about it off-line and agreed to move it to a different branch.

File: vsts_ci/azure-pipelines-perf.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    steps: 
+    - template: continuous-perf.yml
+    dependsOn: SetupScript
+    condition: succeeded()
+
+  - job: RaspberryPiPerfTests
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - template: continuous-perf.yml
```
[the remark this answers is not in the document]
Comment (mhshami01) [reply vsts_ci/azure-pipelines-perf.yml:43]: discussed offline and agreed to leave as is.

**Pull request merged**